clang/lib/APINotes/APINotesFormat.h

```diff
@@ -24,7 +24,7 @@ const uint16_t VERSION_MAJOR = 0;
 /// API notes file minor version number.
 ///
 /// When the format changes IN ANY WAY, this number should be incremented.
-const uint16_t VERSION_MINOR = 33; // SwiftEscapable
+const uint16_t VERSION_MINOR = 34; // SwiftEscapable
```

Comment:
Please update this comment to indicate the format change, e.g. `// SwiftReturnOwnership`